test/support/stubs/AppConfigStub.scala

```diff
@@ -22,6 +22,7 @@ import play.api.mvc.RequestHeader
 import support.utils.TaxYearUtils.taxYearEOY
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
+//TODO Fix Me : convert AppConfig to Trait and define implementation
```

Comment:
Can we create a ticket for this so it doesn't get forgotten (or perhaps a small epic to align the usage of AppConfig across our repos to use the same trait -> implemented class pattern)